PCbuild/pyexpat.vcxproj

```diff
@@ -61,16 +61,20 @@
       <AdditionalIncludeDirectories>$(PySourcePath)Modules\expat;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
       <PreprocessorDefinitions>_CRT_SECURE_NO_WARNINGS;PYEXPAT_EXPORTS;XML_STATIC;%(PreprocessorDefinitions)</PreprocessorDefinitions>
     </ClCompile>
+    <Link>
+      <AdditionalDependencies>$(expatLibDir)\libexpat.lib</AdditionalDependencies>
+      <BaseAddress>0x1e000000</BaseAddress>
```

Comment:
I don't think the Base Address's are still needed with the new compiler -> though I'm not sure - I think I looked at removing these altogether, but never was confident in the change.

Reply:
Yeah, I think the last time I dealt with it, I just changed the base address to be something that made the compiler happy. Should I change the base address?

Reply:
@ucodery any opinion on this?

Reply:
Errrr, Last time I made a patch involving BaseAddress I am pretty sure I backported the exact address value from the upstream file content. If we don't need it, great. But not sure what address would be best practice.

Reply:
Let's leave it for now then, and if we remove the base addresses we'll do it as a separate change.